zng/string.go

```diff
@@ -23,7 +23,7 @@ func EncodeString(s string) zcode.Bytes {
 
 func DecodeString(zv zcode.Bytes) (string, error) {
 	if zv == nil {
-		return "", ErrUnset
+		return "", nil
 	}
 	return string(zv), nil
```

Comment:
Nit: No need for the if statement; `return string(zv), nil` alone is sufficient.